config/packages/yiisoft/view/params.php

```diff
@@ -1,20 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-use Yiisoft\Assets\AssetManager;
-use Yiisoft\Definitions\Reference;
-
-return [
-    'yiisoft/view' => [
-        'basePath' => '@views',
```

Comment:
That would not be set from the package defaults. Are you sure we don't need it?

Reply:
Not used